### PR TITLE
Orientation Form :added enum in orientation and removed hard codded value

### PR DESCRIPTION
**Pull request by nyb12**

### CODE REVIEW

File: apps/front-end/public/locales/hi/translation.json [hunk 1]
```diff
@@ -468,9 +468,6 @@
   "RELATION": "नाता",
   "REMARKS": "टिप्पणियां",
   "REMINDER": "अनुस्मारक",
-  "REMINDERS_1_DAY_BEFORE": "1 दिन पहले",
```
Comment (manojLondhe) [inline apps/front-end/public/locales/hi/translation.json:471]: why are these removed?

Comment (nyb12) [reply apps/front-end/public/locales/hi/translation.json]: it was conflicts issue resolved ,added again

File: apps/front-end/src/pages/front-end/orientation/Orientation.js [hunk 2]
```diff
@@ -92,6 +95,52 @@ export default function Orientation({
...
+        ["properties"]: {
+          ...schema["properties"],
+          [key]: {
+            ...properties,
+            ...(_.isEmpty(arr) ? {} : enumObj),
+          },
+        },
+      };
+    }
+  };
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/orientation/Orientation.js:142]: use helper function `getOption()`
